hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/RawRecord.scala

```diff
@@ -43,6 +46,10 @@ case class RawRecord(id: Long, fields: Map[String, Array[String]]) {
   /** Get the singleton value for an attribute in this record if one exists, parsed as a boolean. */
   def getOptionalBoolean(field: String): Option[Boolean] = getOptional(field).map(_ == "1")
 
+  /** Get the singleton value for an attribute in this record if one exists, parsed as a date. */
+  def getOptionalDate(field: String): Option[LocalDate] =
+    getOptional(field).map(LocalDate.parse(_, DateTimeFormatter.ofPattern("MM-dd-yyyy")))
```

Comment:
Need to check some example data to be sure this date format will actually work.

Reply:
Weirdness... I can't actually find any examples of date files in extraction outputs. It's been a few weeks since I ran it so things might have updated in RedCap, @snwessel have you seen any examples IRL?